### ISSUE TITLE
Cannot import an inteface under a different name

### ISSUE THREAD

**Issue by michwill** [vyperlang/vyper#1753]
### Version Information

* vyper Version (output of `vyper --version`): 0.1.0b14

### What's your issue about?

When importing module under a different name, e.g.:
```python
import module_a as module_b
```
However, this causes an error when compiling the contract:
```
E                       vyper.exceptions.StructureException: line 1:0 Unknown interface module_b
E                       ---> 1 import module_a as module_b
E                       -------^
E                            2
```

What works is only `import module_a as module_a`

**Comment by iamdefinitelyahuman** [comment]
I haven't been able to reproduce the behavior.  What command are you using to run the compiler?

**Comment by iamdefinitelyahuman** [comment]
I think this was addressed by #2083
